### ISSUE TITLE
Overwrote update of PR #402

### ISSUE THREAD

**Issue by klesh** [merico-dev/lake#420]
## Describe the bug
Some update of PR #402 got overwrote along the way, I fixed some easy ones. Here I would list the other issues I found on main branch. 

1. All settings should have some kind of helper link to the document on our repo, Most of them are section-wise
![image](https://user-images.githubusercontent.com/61080/133871912-5b31b8d3-3932-493c-905b-e119f79b9d4c.png)
2. The order of the menu doesn't seem right, maybe we can arrange them in time order, like, Configuration/Plugins/Trigger
![image](https://user-images.githubusercontent.com/61080/133871946-871f5889-adac-4ff6-9645-6468bc57bc1d.png)
3. `Collection Plugins` should be renamed to `Plugins` since they are not only for collection for now, nor for the future.
4. The hinting message to tell user don't change DB_URI/HOST/PORT was gone
![image](https://user-images.githubusercontent.com/61080/133872089-9549c59a-2240-4389-83f2-43ae8d8e1c01.png)

**Comment by oddscenes** [comment]
@klesh All good to me except point 2. Adding triggers button below the plugins list is kind of weird placement. Will be ok right now with 3 plugins but would be lost when theres 10+ plugins. The user wouldn't see the trigger button. 

Structurally its current setup which I think we should keep:
- admin pages (config/trigger etc)
- plugin pages

**Comment by klesh** [comment]
@oddscenes Good point. For plugins being too many, we could collapse the plugins menu once they were all configured. So, for the first time setup, user go though all plugins (maybe there should be a page to enable/disable plugins they like). Then plugins menu become collapsed by default from that point forward.
@Startrekzky  check this out, we need some opinion input from Product.

**Comment by Startrekzky** [comment]
Hi @oddscenes and @klesh 
My proposal is moving "Jira", "Gitlab", "Jenkins" tabs to the right as anchor links. Two benefits:

1. The height of the left tab bar won't be affected by the number of supported data sources. The left bar will always be "Configuration", "Collection Plugin" and "Triggers". The problem you were discussing will be solved.
2. All configuration is aggregated into the same page so that users 
a) don't have to switch tabs when configuring a new plugin
b) don't have to switch tabs to check if all fields have been filled
c) are less likely to forget to configure certain data sources

![image](https://user-images.githubusercontent.com/14050754/134349289-3c02d617-4eac-4491-9c0b-8ed925901904.png)
